flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/catalog/PostgresCatalogTestBase.java

```diff
@@ -81,14 +82,16 @@ public static void init() throws SQLException {
 		createTable(PostgresTablePath.fromFlinkTableName(TABLE4), getSimpleTable().pgSchemaSql);
 		createTable(PostgresTablePath.fromFlinkTableName(TABLE5), getSimpleTable().pgSchemaSql);
 
-		executeSQL(DEFAULT_DATABASE, String.format("insert into public.%s values (1);", TABLE1));
-
-		// table: testdb.public.t2
-		// table: testdb.test_schema.t3
-		// table: testdb.public.datatypes
+		// table: test.public.t2
+		// table: test.test_schema.t3
+		// table: postgres.public.dt
+		// table: postgres.public.dt2
```

Comment:
do we have created this table `postgres.public.dt2`?

Reply:
removed for now